### PR TITLE
Updates Helper text to be correct

### PR DESCRIPTION
**Pull request by belcherj**
Updates copy to be more useful and remove collaborator link.

### CODE REVIEW

File: includes/admin/helper/class-wc-helper.php [hunk 1]
```diff
@@ -286,7 +286,7 @@ public static function render_helper_output() {
 				$data['_actions'][] = $action;
 			} else {
 				$action = array(
-					'message' => sprintf( __( 'To receive updates and support for this extension, you need to <strong>purchase</strong> a new subscription or <a href="%1$s" target="_blank">be added as a collaborator</a>.', 'woocommerce' ), 'https://docs.woocommerce.com/document/adding-collaborators/' ),
+					'message' => sprintf( __( 'To receive updates and support for this extension, you need to <strong>purchase</strong> a new subscription or consolidate your extensions to one connected account by <strong>sharing or transferring</strong> this extension to this connected account.', 'woocommerce' ), 'https://docs.woocommerce.com/document/adding-collaborators/' ),
```
Comment (dougaitken) [inline includes/admin/helper/class-wc-helper.php]: Did you mean to link to this - https://docs.woocommerce.com/document/activating-upgrading-transferring-licenses/ @belcherj

**Comment by garymurray** [comment]
We can link to this page: https://docs.woocommerce.com/document/managing-woocommerce-com-subscriptions/

Or specically:

Sharing: https://docs.woocommerce.com/document/managing-woocommerce-com-subscriptions/#section-10

Transfers: https://docs.woocommerce.com/document/managing-woocommerce-com-subscriptions/#section-5

**Comment by belcherj** [comment]
@garymurray links added, please re-review.  TY

**Review by garymurray — APPROVED** [review]

**Comment by kloon** [comment]
@belcherj can you please use the sprintf variable to pass through the url and not put the URL directly in the string? As it were previously.

**Comment by belcherj** [comment]
@kloon done.

**Pull request merged**